sql/core/src/test/scala/org/apache/spark/sql/SparkSessionBuilderSuite.scala

```diff
@@ -17,13 +17,15 @@
 
 package org.apache.spark.sql
 
+import org.scalatest.BeforeAndAfterEach
+
 import org.apache.spark.{SparkConf, SparkContext, SparkFunSuite}
 import org.apache.spark.sql.internal.SQLConf
 
 /**
  * Test cases for the builder pattern of [[SparkSession]].
  */
-class SparkSessionBuilderSuite extends SparkFunSuite {
+class SparkSessionBuilderSuite extends SparkFunSuite with BeforeAndAfterEach {
 
   private var initialSession: SparkSession = _
```

Comment:
Seems this is only used in the first test case? shall we just move it to the first test case?